src/MainnetController.sol

```diff
@@ -41,18 +42,23 @@ interface IPSMLike {
 
 contract MainnetController is AccessControl {
 
-    // TODO: Inherit and override interface
-
     /**********************************************************************************************/
     /*** State variables                                                                        ***/
     /**********************************************************************************************/
 
     bytes32 public constant FREEZER = keccak256("FREEZER");
     bytes32 public constant RELAYER = keccak256("RELAYER");
 
+    bytes32 public constant LIMIT_USDS_MINT    = keccak256("LIMIT_USDS_MINT");
+    bytes32 public constant LIMIT_USDS_BURN    = keccak256("LIMIT_USDS_BURN");
+    bytes32 public constant LIMIT_USDS_TO_USDC = keccak256("LIMIT_USDS_TO_USDC");
+    bytes32 public constant LIMIT_USDC_TO_USDS = keccak256("LIMIT_USDC_TO_USDS");
+    bytes32 public constant LIMIT_USDC_TO_CCTP = keccak256("LIMIT_USDC_TO_CCTP");
+
     address public immutable buffer;
 
     IALMProxy    public immutable proxy;
+    IRateLimits  public immutable rateLimits;
```

Comment:
While we will frequently update the controllers, I don't think it's likely we will update the rate limiter contract. Can you think of why this would be?

Even if it happens on the odd occasion we can redeploy the controller which we want to keep as stateless as possible imo.

Reply:
Ok, yeah, maybe that's not gonna be the case all that often and it's better to swap the whole controller.